prime/src/main/java/com/telenordigital/prime/disruptor/PrimeEvent.java

```diff
@@ -17,13 +17,22 @@
     private String msisdn;
 
     /**
-     *  Origin of word 'bucket' - PGw consumes data in `buckets` of 10 MB ~ 100 MB at a time
-     *  This field is used in...
-     *  1. Request to fetch bucket
-     *  2. Response of actual allowed bucket
-     *  3. Request to return unused bucket
+     *  Origin of word 'bucket' - P-GW consumes data in `buckets` of 10 MB ~ 100 MB at a time
+     *  This field is used in.
+     *  Request to reserve a new bucket of bytes
      */
-    private long bucketBytes;
+    private long requestedBucketBytes;
```

Comment:
Originally, the PrimeEvent was a generic event handled by Prime. Now, it is heavily inclined towards being a PGW request and not much of Topup or Get Balance. Current changes would not break anything. But we should refactor this in future.